### PR TITLE
feat(apps/discord-bot): add reaction to good morning messages

### PR DESCRIPTION
**Pull request by durmusrasit**

### CODE REVIEW

File: apps/discord-bot/config.ts [hunk 1]
```diff
@@ -1,2 +1,10 @@
 // Discord Channel Ids string[]
 export const TWITCH_EVENTS_CHANNELS = [""];
+export const GOOD_MORNING_STRINGS = [
+  "günaydı",
+  "gunaydi",
+  "günaydi",
+  "gunaydi",
+  "gunaydı",
+];
```
Comment (usirin) [inline apps/discord-bot/config.ts:9]:
```suggestion
];

const GOOD_MORNING_REGEXP = /g(ü|u)nayd(i|ı)n/
export const isGoodMorningMessage = (msg: string): boolean => GOOD_MORNING_REGEXP.test(msg)
```

File: apps/discord-bot/config.ts [hunk 2]
```diff
@@ -1,2 +1,10 @@
 // Discord Channel Ids string[]
```
Comment (usirin) [inline apps/discord-bot/config.ts:1]:
```suggestion
import { reactMessageWithEmoji } from "./utils/reactionHelpers";

// Discord Channel Ids string[]
```

File: apps/discord-bot/config.ts [hunk 3]
```diff
@@ -1,2 +1,10 @@
...
 export const TWITCH_EVENTS_CHANNELS = [""];
+export const GOOD_MORNING_STRINGS = [
+  "günaydı",
+  "gunaydi",
+  "günaydi",
+  "gunaydi",
+  "gunaydı",
+];
+export const SUN_WITH_FACE_EMOJI = "🌞";
```
Comment (usirin) [inline apps/discord-bot/config.ts:10]:
```suggestion
const SUN_WITH_FACE_EMOJI = "🌞";
export const reactWithSun = (reaction: MessageReaction | PartialMessageReaction) =>
  reactMessageWithEmoji(reaction, SUN_WITH_FACE_EMOJI)
```

File: apps/discord-bot/events/messageReactionAdd.ts [hunk 4]
```diff
@@ -6,13 +6,15 @@ import {
 } from "discord.js";
 import { selectByNameCallback } from "../utils/helpers";
 import { addRole } from "../utils/memberFunctions";
+import { GOOD_MORNING_STRINGS, SUN_WITH_FACE_EMOJI } from "../config";
```
Comment (usirin) [inline apps/discord-bot/events/messageReactionAdd.ts:9]:
```suggestion
import { isGoodMorningMessage, reactWithSun } from "../config";
```

File: apps/discord-bot/events/messageReactionAdd.ts [hunk 5]
```diff
@@ -21,6 +23,13 @@ export default {
       .REACTION_RULES_MESSAGE_STARTSWITH as string;
 
     const messageContent = reaction.message.content as string;
+
+    for (const gmString of GOOD_MORNING_STRINGS) {
+      if (messageContent.startsWith(gmString)) {
```
Comment (usirin) [inline apps/discord-bot/events/messageReactionAdd.ts:28]:
```suggestion
      if (isGoodMorningMessage(gmString)) {
```

File: apps/discord-bot/events/messageReactionAdd.ts [hunk 6]
```diff
@@ -21,6 +23,13 @@ export default {
       .REACTION_RULES_MESSAGE_STARTSWITH as string;
 
     const messageContent = reaction.message.content as string;
+
+    for (const gmString of GOOD_MORNING_STRINGS) {
+      if (messageContent.startsWith(gmString)) {
+        reactMessageWithEmoji(reaction, SUN_WITH_FACE_EMOJI);
```
Comment (usirin) [inline apps/discord-bot/events/messageReactionAdd.ts:29]:
```suggestion
        reactMessageSun(reaction);
```

File: apps/discord-bot/events/messageReactionAdd.ts [hunk 7]
```diff
@@ -6,13 +6,15 @@ import {
 } from "discord.js";
 import { selectByNameCallback } from "../utils/helpers";
 import { addRole } from "../utils/memberFunctions";
+import { GOOD_MORNING_STRINGS, SUN_WITH_FACE_EMOJI } from "../config";
+import { reactMessageWithEmoji } from "../utils/reactionHelpers";
```
Comment (usirin) [inline apps/discord-bot/events/messageReactionAdd.ts:10]:
```suggestion
```

**Review by usirin — COMMENTED** [review]
Made a couple of improvements, especially around how we match a message to a "gunaydin" message and it's variants. Let me know if it doesn't make sense

**Review by mertsabinov — APPROVED** [review]

**Pull request merged**